src/misc.asm

```diff
@@ -32,6 +36,22 @@ org $828BB3
     JSL gamemode_end
 
 
+; Replace unnecessary logic checking controller input to toggle debug CPU brightness
+; with logic to collect the v-counter data
+org $828AB1
+    %a8() : LDA $4201 : ORA #$80 : STA $4201 : %ai16()
+    LDA $2137 : LDA $213D : STA !ram_vcounter_data
+
+    ; For efficiency, re-implement the debug brightness logic here
+    LDA $0DF4 : BEQ .skip_debug_brightness
```

Comment:
This logic is reimplemented because it was moved up.  The previous overwritten logic would first check debug mode, and if in debug mode then check if the debug shortcut for brightness was input.  That's removed since we can just turn on debug CPU brightness through the menu.

Skipping the initial debug mode check saves 7 CPU cycles every frame